GENERATED_README.md

```
@@ -0,0 +1,453 @@
+# IPB CLI Command Reference
+
+> This documentation is auto-generated from the CLI command definitions.
+> Last generated: 2025-11-05T06:34:44.819Z
+
+## Table of Contents
+
+- [cards](#cards)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)